data/blog/electron-4-0.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Electron 4.0.0
+author: BinaryMuse
+date: '2018-12-18'
+---
+
+The Electron team is excited to announce that the stable release of Electron 4 is now
+available from [electronjs.org](https://electronjs.org/) and via `npm install electron@latest`! It's jam-packed with upgrades, fixes, and new features, and we can't wait to see what you build with them. Below are details about this release, and we welcome your feedback as you explore.
+
+---
+
+## App Feedback Program
+
+The [App Feedback Program](https://electronjs.org/blog/app-feedback-program) we instituted during the development of Electron 3.0 was successful, so we've continued it during the development of 4.0 as well. We'd like to extend a massive thank you to Atlassian, Atom, Microsoft Teams, Oculus, OpenFin, Slack, Symphony, VS Code, and other program members for their involvement in the program during the 4.0 beta cycle. To learn more about the App Feedback Program and to participate in future betas, check out [our blog post about the program](https://electronjs.org/blog/app-feedback-program).
```

Comment:
@sofianguy I pulled this list from the 3.0 blog post; is it accurate for the 4.0 run of the AFP?

Reply:
The link to the program blog post is correct 👍 

I think you're referring to the list of company participants? It's not correct for 4.0. I can ask company participants about listing their name; I need a couple days. Or we can just sub with "program participants" or the like

Reply:
@sofianguy I think it'd be rad to list the companies if that's something we can do, otherwise I can drop the list and just use the generic "program participants."

Reply:
okie dokie. Give me a couple days and i'll update you with the list of company names who approve

Reply:
approval from companies: Atlassian, Discord, WhatsApp, Slack, Symphony, MS Teams